### PR TITLE
fix(plugin-upload): size limit comparison issue

### LINKED ISSUE #5233: Size Limit comparaison issue
when uploading a file even if its size is less then the size the limit the response's status was 400
and it was due to the size limit
file size = 853.40
limit size =  5000000000

if i changed the code to parse the tow numbers it will work correctly

**Code snippets**
if (parseFloat(file.size) > parseFloat(config.sizeLimit)) {

**System**
- Node.js version: 10.15.0
- Strapi version: 3.0.0-beta.17.5
- Database: mongodb
- Operating system: linux

### PR DESCRIPTION
**Pull request by 7mioda**
#### Description of what you did:
Fix #5233

#### My PR is a:

- [ ] 💥 Breaking change
- [x] 🐛 Bug fix
- [ ] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [ ] Documentation
- [ ] Framework
- [x] Plugin

#### Manual testing done on the following databases:

- [ ] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
@7mioda your PR is failing the DCO check: https://github.com/strapi/strapi/pull/5242/checks?check_run_id=443421022

**Comment by alexandrebodin** [comment]
@7mioda Thanks for this PR. Would you mind creating a getConfig method in the service that returns the config with the right values casted ? and update the updateSettings to cast the value before they are saved in DB ? 

this way we can assume the value we get is a float when comparing in this service.

**Comment by 7mioda** [comment]
okey , no problem

File: packages/strapi-plugin-upload/services/Upload.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ module.exports = {
             : '',
         buffer,
         mime: stream.type,
-        size: (stream.size / 1000).toFixed(2),
+        size: parseFloat((stream.size / 1000).toFixed(2)),
```
Comment (alexandrebodin) [inline packages/strapi-plugin-upload/services/Upload.js]: `parseFloat` shouldn't be required here. the value is already casted :)

Comment (7mioda) [reply packages/strapi-plugin-upload/services/Upload.js]: okey ;)

**Pull request merged**